src/main/java/com/robinpowered/sdk/credential/AccessTokenCredential.java

```diff
@@ -23,10 +23,18 @@
      */
 
     public AccessTokenCredential(String token) {
+        if (token == null) {
+            throw new IllegalArgumentException("Credentials cannot have a null values.");
+        }
+
         this.token = token;
     }
 
     public AccessTokenCredential(Token token) {
```

Comment:
Skipping nonnull annotations, since it requires a 3rd party dependency imported.